src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -0,0 +1,237 @@
+                placeholder="Enter change request ID..."
+                errorMessage={errors.crId}
+                type="number"
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <FormControl>
+              <FormLabel>Project WBS Number</FormLabel>
+              <ReactHookTextField
+                name="wbsNum"
+                control={control}
+                placeholder="Enter project WBS number..."
+                errorMessage={errors.wbsNum}
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <Controller
+              name="startDate"
+              control={control}
+              rules={{ required: true }}
+              render={({ field: { onChange, value } }) => (
+                <FormControl>
+                  <FormLabel>Start Date (YYYY-MM-DD)</FormLabel>
+                  <DatePicker
+                    inputFormat="yyyy-MM-dd"
+                    onChange={onChange}
+                    className={'padding: 10'}
+                    value={value}
+                    renderInput={(params) => <TextField autoComplete="off" {...params} />}
+                  />
+                </FormControl>
+              )}
+            />
+          </Grid>
+          <Grid item xs={12} md={2}>
+            <FormControl>
+              <FormLabel>Duration</FormLabel>
+              <ReactHookTextField
+                name="duration"
+                control={control}
+                placeholder="Enter duration..."
+                errorMessage={errors.duration}
+                type="number"
+                endAdornment={<InputAdornment position="end">weeks</InputAdornment>}
+              />
+            </FormControl>
+          </Grid>
+        </Grid>
+        <Grid container spacing={2} direction="column" sx={{ mt: 1 }}>
+          <Grid item xs={12} md={2}>
+            <FormControl>
+              <FormLabel>Dependencies</FormLabel>
+              {dependencies.map((_element, i) => {
+                return (
+                  <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+                    <TextField
+                      required
+                      autoComplete="off"
+                      {...register(`dependencies.${i}.wbsNum`)}
+                      sx={{ width: 9 / 10 }}
+                    />
+                    <IconButton type="button" onClick={() => removeDependency(i)} sx={{ mx: 1, my: 0 }}>
+                      <DeleteIcon />
+                    </IconButton>
+                  </Grid>
+                );
+              })}
+              <Button
+                variant="contained"
+                color="success"
+                onClick={() => appendDependency({ wbsNum: '' })}
+                sx={{ my: 2, width: 'max-content' }}
+              >
+                + ADD NEW DEPENDENCY
+              </Button>
+            </FormControl>
```

Comment:
Can you put the whole dependencies FormControl into a helper, I think it's long enough to warrant it/ it'll help make this long file be a bit more readable